package.json

```diff
@@ -19,18 +19,20 @@
     "!test.js"
   ],
   "dependencies": {
-    "file-type": "<=16.5.1",
+    "file-type": "^18.2.1",
     "multiblob-http": "^1.2.0",
     "pull-box-stream": "^1.0.13",
     "pull-stream": "^3.3.0"
   },
   "devDependencies": {
+    "pull-file": "^1.1.0",
+    "scuttle-testbot": "^1.11.0",
     "ssb-blobs": "^1.2.2",
-    "ssb-server": "^15.2.0",
```

Comment:
oh yeah `ssb-server` was also bringing in a lot of old dependencies which were polluting the audit output